homeassistant/components/plex/config_flow.py

```diff
@@ -244,15 +248,16 @@ def __init__(self):
         session = async_get_clientsession(self.hass)
         self.plexauth = PlexAuth(payload, session)
         await self.plexauth.initiate_auth()
-        auth_url = self.plexauth.auth_url()
-        self.hass.helpers.event.async_call_later(
-            DELAY_CONFIGURE, self.hass.config_entries.flow.async_configure(self.flow_id)
+        base_url = self.hass.config.api.base_url.rstrip("/")
```

Comment:
The strip is already done by the core.

https://github.com/home-assistant/home-assistant/blob/8c01ed8a1ff92f9b00018b870025de67307d18ed/homeassistant/components/http/__init__.py#L132-L141